### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
Add Javascript classes for Displaying and Storing Information.
Add Methods Inside classes To add and remove items.
Implement Local Storage to Store and Display Data.

### CODE REVIEW

File: javascriptclasses.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+class Book { // eslint-disable-line
```
Comment (Emmyn5600) [inline javascriptclasses.js:1]: - [ ] Don't disable eslint errors, kindly fix them without disabling them.

File: javascriptclasses.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(key) {
+    const books = Store.getBooksFromStorage();
+    books.forEach((book, i) => {
+      if (book.key == key) { // eslint-disable-line
+        books.splice(i, 1);
+      }
```
Comment (Emmyn5600) [inline javascriptclasses.js:32]: - [ ] Don't disable eslint errors, kindly fix them without disabling them.

File: javascriptclasses.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+document.querySelector('#book-form').addEventListener('submit', () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const key = Math.floor(Math.random() * 100000000);
+
+  const book = new Book(title, author, key);
+
+  Display.addBookToDisplay(book);
+  Store.addBookToStorage(book);
+});
```
[the remark this answers is not in the document]
Comment (Danieldotcomcoder) [reply javascriptclasses.js]: Thank you . I fix this issue.

[the remark this answers is not in the document]
Comment (henatan99) [reply javascriptclasses.js]: Well done! This issue is resolved! :clap: :+1: :+1:

**Pull request merged**